public/stylesheets/netlogoweb.css

```diff
@@ -1,32 +1,61 @@
+/*
+ * OpenSans
+ * */
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300..800;1,300..800&display=swap');
```

Comment:
Not legit.  NetLogo Web needs to bundle everything that it uses and to be usable without an internet connection.